File: Python/car.py
from account import Account

class Car:
    id = int
    license = str
    driver = Account("","")
    __passenger = int
    
    def __init__(self, license, driver):
        self.license = license
        self.driver = driver
    
    # No obligatorio pero pos qué carajos 
    def printDataCar (self):
        print("Driver: " + self.driver.name + "\nDrivers license: " + self.driver.document + "\nCar license: " + self.license)
    
    def getPassenger (self):
        return self.__passenger
    
    def setPassenger (self, passenger):
        if(passenger == 4):
            self.__passenger = passenger
        else:
            raise ValueError(f'El carro debe ser de al menos 4 pasajeros')

File: JS/car.js
class Car {
    constructor (license, driver, passenger){
        this.id;
        this.license = license;
        this.driver = driver;
        this.#passenger = passenger;
    }
    
    #passenger = 0;
    get passenger () {
        return this.#passenger;
    }
    set passenger (passenger) {
        passenger == 4 ? this.#passenger = passenger 
        : console.error('El carro debe ser de al menos 4 pasajeros');
    }

    printDataCar = (brand, model) => {
        if (brand && model && this.#passenger == 4){
            console.log(`Driver: ${this.driver.name} \nDriver document: ${this.driver.document} \nCar license: ${this.license} \nBrand: ${brand} \nModel: ${model} \nPassengers: ${this.#passenger}`);
        } else if (this.#passenger != 4){
            this.#passenger = undefined;
        }
    }
}